File: 删除有序数组中的重复项.c
int removeDuplicates(int* nums, int numsSize) {
    int n = 1;

    if(numsSize == 0) return 0; //如果數組爲空， 直接返回0

    for(int i = 1; i < numsSize; i++){
        if(nums[n - 1] != nums[i]){ //如果當前元素與前一個元素不相同，則將當前元素放到新數組的末尾
            nums[n++] = nums[i];
        }
    }

    return n;
}

File: Bypyhton/删除有序数组中的重复项.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0 
        n = 1
        while n < len(nums):
            if nums[i] != nums[n]:
                i += 1
                nums[i] = nums[n]
            n += 1
        return i + 1